src/components/TimePickerInput/index.jsx

```diff
@@ -85,9 +84,18 @@ const TimePickerInput = forwardRef(
       return true;
     };
 
+    const onAmPmChange = () => {
+      const value = valueRef.current;
```

Comment:
Do we need this? Does the value we get from the useRefState have the same value?

Reply:
Yes we need this. this function is called from dom eventlistener. ref value and state value are same, but dom eventlistener doesn't get the latest state value.